virttest/remote.py

```diff
@@ -201,6 +201,71 @@ def handle_prompts(session, username, password, prompt, timeout=10,
     return output
 
 
+def handle_v2v_prompts(session, username, password, timeout=10,
+                       debug=False, input_line=None):
+    """
+    Connect to a virt-v2v conversion session.
+
+    Wait for questions and provide answers.  If timeout expires while
+    waiting for output from the child (e.g. a password prompt or
+    a shell prompt) -- fail.
+
+    :param session: The virt-v2v session instance to operate on
+    :param username: The username to send in reply to a login prompt
+    :param password: The password to send in reply to a password prompt
+    :param timeout: The maximal time duration (in seconds) to wait for each
+            step of the login procedure (i.e. the "Are you sure" prompt, the
+            password prompt, the shell prompt, etc)
+    :raise LoginTimeoutError: If timeout expires
+    :raise LoginProcessTerminatedError: If the client terminates during login
+    :raise LoginError: If some other error occurs
+    :return: If connect succeed return the output text to script for further
+             debug.
+    """
+
+    output = ""
+    while True:
+        time.sleep(1)
```

Comment:
this can be avoided, unless it has some specific reason

Reply:
Updated, thanks